sandbox/builder/src/main/java/org/kaaproject/kaa/sandbox/AbstractSandboxBuilder.java

```diff
@@ -344,6 +344,10 @@ protected void initBoxData() throws Exception {
             }
         }
 
+        LOG.info("Executing Cassandra cql script...");
```

Comment:
We need to control AbstractSandboxBuilder behavior using arguments. In other words, we need a dedicated argument that will trigger either Cassandra setup script or Mongo DB or both (all).